web/src/components/PresetsModal/PresetsModal.scss

```diff
@@ -8,6 +8,9 @@
 
 .presets-modal-card {
   max-height: 75vh;
+  width: 90vw;
+  max-width: 30rem;
+  text-overflow: wrap;
```

Comment:
Gave it the same text wrapping and width limits as the streams and zones modals so that long names don't do this:
![image](https://github.com/user-attachments/assets/fae6f5c9-c14c-436b-b4b2-773b199d5d9a)